docs/sources/setup-grafana/configure-security/secret-scan.md

```diff
@@ -0,0 +1,88 @@
+---
+description: Detect and revoke leaked Grafana service account tokens
+labels:
+  products:
+    - cloud
+    - enterprise
+    - oss
+title: Configure Grafana secret scanning
+menuTitle: Configure secret scanning
+weight: 1000
+---
+
+# Configure Grafana secret scanning
+
+> **Note:** Available from Grafana 10.0.
+
+With Grafana, you can use the GitHub Secret Scanning service to determine if your [service account tokens]({{< relref "../../administration/service-accounts/" >}}) have been leaked on GitHub.
+
+<div class="clearfix"></div>
+
+When GitHub Secret Scanning detects a Grafana secret, its hash is stored in Grafana's secret scanning service.
+
+Grafana instances, whether on-premises or on the cloud, can use this service to verify if a token generated by the instance has been made public. This verification is done by comparing the token's hash with the exposed token's hash.
+
+If the service detects a leaked token, it immediately revokes it, making it useless.
+
+> **Note:** If the `revoke` option is disabled, the service only sends a notification to the configured webhook URL and logs the event. The token is not automatically revoked.
+
+By default, when the Grafana secret scanning service detects a leaked token,
+the service automatically revokes the token and logs the event.
+Additionally, the service can be configured to send an outgoing webhook notification to a webhook URL.
+
+The notification includes a JSON payload that contains the following data:
+
+```json
+{
+  "alert_uid": "c9ce50a1-d66b-45e4-9b5d-175766cfc026",
+  "link_to_upstream_details": <URL to token leak>,
+  "message": "Token of type grafana_service_account_token with name
+sa-the-toucans has been publicly exposed in <URL to token leak>.
+Grafana has revoked this token",
+  "state": "alerting",
+  "title": "SecretScan Alert: Grafana Token leaked"
+}
+```
+
+> secret scanning is disabled by default. No outgoing connections are made until you enable it.
+
+## Before you begin
+
+- Ensure all your API keys have been migrated to service accounts.
+  If you need help with this step, follow [Migrate API keys to Grafana service accounts]({{< relref "../../administration/api-keys/#migrate-api-keys-to-grafana-service-accounts" >}}).
+- Deployed Grafana 10.0 or later.
+
+## Steps
```

Comment:
```suggestion
## Configure secret scanning
```

Reply:
just an fyi, I and changing this back to the name of the task because I realize now that this topic actually contains two tasks, so it is odd to have one called Steps and the other called "Configure outgoing webhook notifications"